### ISSUE TITLE
Drop support for anything before macOS 10.13

### ISSUE THREAD

**Issue by karbassi** [michaelvillar/timer-app#68]
[Apple stopped supporting 10.12 and earlier.](https://en.wikipedia.org/wiki/MacOS_Sierra#Release_history)

Should we stop supporting anything less than macOS 10.13?

This would allow us to write cleaner code since there has been a lot of new code features in 10.13 and above.

I'd like to hear everyone's views.

**Comment by robinstewart** [comment]
This makes sense to me. We can link to a build of version 1.5 for anyone who needs compatibility going further back.

**Comment by karbassi** [comment]
I agree. I think version 1.5 will be the last supported release of macOS 10.13.
